lib/srv/desktop/windows_server.go

```diff
@@ -523,7 +524,14 @@ func (s *WindowsService) initializeLDAP() error {
 	if err != nil {
 		s.mu.Lock()
 		s.ldapInitialized = false
-		s.scheduleNextLDAPCertRenewalLocked(windowsDesktopServiceCertRetryInterval)
+
+		// failures due to timeouts might be transient, so retry more frequently
+		retryAfter := windowsDesktopServiceCertRetryInterval
+		if errors.Is(err, context.DeadlineExceeded) {
+			retryAfter = 10 * time.Second
```

Comment:
Should we, similar to handling error and retry period above, divide `windowsDesktopServiceCertRetryInterval` by some number to base more frequent interval on it? Or create another const for this interval?